### ISSUE TITLE
failed frame localization logs can't be found

### ISSUE THREAD

**Issue by David-Baddeley** [python-microscopy/python-microscopy#61]
**[Original report](https://bitbucket.org/david_baddeley/python-microscopy/issue/61) by Andrew Barentine (Bitbucket: [AndrewBarentine](https://bitbucket.org/AndrewBarentine), ).**

----------------------------------------

Links displayed by clusterUI for failed localization frames currently point to non-existent files.
```
#!python

Traceback:

File "/home/ubuntu/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  149.                     response = self.process_exception_by_middleware(e, request)

File "/home/ubuntu/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  147.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/ubuntu/python-microscopy/clusterUI/clusterbrowser/views.py" in file
  16.         return HttpResponse(clusterIO.getFile(filename, use_file_cache=False), content_type='')

File "/home/ubuntu/anaconda2/lib/python2.7/site-packages/PYME/IO/clusterIO.py" in getFile
  580.         raise IOError("Specified file could not be found: %s" % filename)

Exception Type: IOError at /files/download/LOGS/rules/000002-84a31f60dcb04ee0b9904da61d012268.log
Exception Value: Specified file could not be found: LOGS/rules/000002-84a31f60dcb04ee0b9904da61d012268.log
```

**Comment by David-Baddeley** [comment]
**Original comment by David Baddeley (Bitbucket: [david_baddeley](https://bitbucket.org/david_baddeley), ).**

----------------------------------------

The unified logging isn't implemented for localizations yet, only recipes.

To resolve this, we need to make line 316 of taskWorkerHttp.py more like line 369.

i.e. returning a `TaskError` instance rather than `None`

It should be a really simple fix, but as failure handling for recipes is a little odd at the moment (things end up getting retried lots of times) it might be worth taking the time to do a bit of testing with this change prior to integrating it.

**Comment by David-Baddeley** [comment]
**Original comment by David Baddeley (Bitbucket: [david_baddeley](https://bitbucket.org/david_baddeley), ).**

----------------------------------------

tentative fix in commit #4896
